components/post_view/post_list_virtualized/post_list_virtualized.jsx

```diff
@@ -611,23 +611,23 @@ export default class PostList extends React.PureComponent {
 
                                         <DynamicSizeList
                                             ref={this.listRef}
+                                            innerRef={this.postListRef}
```

Comment:
Could you try to avoid making styling changes to files that aren't otherwise being modified by a PR? It makes it hard to tell which parts of the PR are important to be reviewed

Reply:
yep ... that just slipped in since I wanted to compare the props on both and there are a lot of them! Will revert that. SRY!